third_party/move/move-vm/runtime/src/loader/script.rs

```diff
@@ -103,7 +103,7 @@ impl Script {
             });
         }
 
-        let code = script.code.code.clone();
+        let code = script.code.code.iter().map(|b| b.clone().into()).collect();
```

Comment:
Maybe we should have some wrapper `Converter` that takes a slice of bytecode and returns a vector of instructions? Then it can be easier to put all logic there in the future and not update function/scripts sites (though we need to add context to this converter so it has to carry module / script info etc.)